### PR TITLE
Hotfix compaction with engineering suggestions

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- titles for all code snippets for consistency

### Staged Changes (Requires MongoDB Corp SSO)

- [Optimize Sync Atlas Usage](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/backend-compaction-hotfix/sync/optimize-sync-atlas-usage/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sync/optimize-sync-atlas-usage.txt [hunk 1]
```diff
@@ -47,6 +47,7 @@ instruction.
    Consider the following {+realm+} history:
 
    .. code-block::
+      :caption: History
```
Comment (andrewlaguna824) [inline source/sync/optimize-sync-atlas-usage.txt:50]: It's not shown here but I think we should be consistent with all the captions for initial state.

E.g. in Advanced Backend Compaction, our initial state is called "Not Compacted" but in the other examples it's called "History". 

There's an inconsistency in captions for Advanced Compacted History vs Advanced Compacted also

Maybe we should use:
- Original History
- Compacted History
- Advanced Compacted History

File: source/sync/optimize-sync-atlas-usage.txt [hunk 2]
```diff
@@ -206,9 +218,29 @@ after any period of time offline -- weeks, months, or even years --
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Advanced backend compaction causes permanent, irreversible changes to
+history. As a result, increasing the client maximum offline time does
+not *immediately* change the length of time before clients experience a
+client reset. Existing history has already been advanced compacted,
+requiring a client reset. New history needs time to accumulate up to
+the new client maximum offline time.
+
+Disabling the client maximum offline time feature will prevent future
```
Comment (dacharyc) [inline source/sync/optimize-sync-atlas-usage.txt:232]: Nit: remove future tense.
```suggestion
Disabling the client maximum offline time feature stops additional
```

File: source/sync/optimize-sync-atlas-usage.txt [hunk 3]
```diff
@@ -206,9 +218,29 @@ after any period of time offline -- weeks, months, or even years --
...
 accumulate many changes. With a large changeset, synchronization
 requires more time and data usage.
+   
+Client Maximum Offline Time Does Not Immediately Influence Client Resets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Advanced backend compaction causes permanent, irreversible changes to
+history. As a result, increasing the client maximum offline time does
+not *immediately* change the length of time before clients experience a
+client reset. Existing history has already been advanced compacted,
```
Comment (dacharyc) [inline source/sync/optimize-sync-atlas-usage.txt:228]: Using "advanced compacted" in this way feels grammatically off to me, but I can't put my finger on why. We've already established that we're talked about advanced compaction. Could we say "changed" here instead?

File: source/sync/optimize-sync-atlas-usage.txt [hunk 4]
```diff
@@ -206,9 +218,29 @@ after any period of time offline -- weeks, months, or even years --
...
+requiring a client reset. New history needs time to accumulate up to
+the new client maximum offline time.
+
+Disabling the client maximum offline time feature will prevent future
+advanced backend compaction, but already advanced compacted history
```
Comment (dacharyc) [inline source/sync/optimize-sync-atlas-usage.txt:233]: Same note as above - using "advanced compacted" here reads odd to me. Can we do something like:

> Disabling... advanced backend compaction. History that has already been modified permanently requires client resets to synchronize.

**Pull request merged**